src/components/elements/types.ts

```diff
@@ -49,11 +53,15 @@ export enum EElementType {
 export type TElementSchema =
 	| IAlertSchema
 	| IDividerSchema
-	| ITextSchema
-	| IWrapperSchema
 	| IGridSchema
+	| IListItemSchema
```

Comment:
remove as `list-item` can only be added as a direct descendant of `ordered-list` and `unordered-list`

Reply:
hmm but this needs to be passed down to `Wrapper` (for cond rend handling), which requires the ui type to be known

Reply:
in such cases, we pass them as `children` so we will need to add `IListItemSchema` typing to the children prop
ref: https://github.com/LifeSG/web-frontend-engine/blob/main/src/components/elements/wrapper/types.ts#L20